collections/specprocessor/crowdsource/controlpanel.php

```diff
@@ -40,26 +42,25 @@
 		<div style="float:right;"><a href="#" onclick="toggle('projFormDiv')"><img src="../../images/edit.png" /></a></div>
 		<div style="font-weight:bold;font-size:130%;"><?php echo (($omcsid && $projArr)?$projArr['name']:''); ?></div>
 		<div>
-			This module can be used to submit and manage records for data entry by the
-			general public. For more information, see the <a href="https://symbiota.org/crowdsourcing-within-symbiota-2/">Symbiota documentation on crowdsourcing</a>.
+			<?php echo $LANG['CROWDSOURCE_EXPLAIN']; ?>
 		</div>
 		<div id="projFormDiv" style="display:none">
 			<fieldset style="margin:15px;">
-				<legend><b>Edit Project</b></legend>
+				<legend><b><?php echo $LANG['EDIT_PROJECT']; ?></b></legend>
 				<form name="csprojform" action="index.php" method="post">
 					<div style="margin:3px;">
-						<b>General Instructions:</b><br/>
+						<b><?php echo $LANG['GENERAL_INSTRUCTIONS']; ?>:</b><br/>
 						<textarea name="instr" style="width:500px;height:100px;"><?php echo (($omcsid && $projArr)?$projArr['instr']:''); ?></textarea>
 					</div>
 					<div style="margin:3px;">
-						<b>Training Url:</b><br/>
+						<b><?php echo $LANG['TRAINING_URL']; ?>:</b><br/>
 						<input name="url" type="text" value="<?php echo (($omcsid && $projArr)?$projArr['url']:''); ?>" style="width:500px;" />
 					</div>
 					<div>
 						<input name="collid" type="hidden" value="<?php echo $collid; ?>" />
 						<input name="omcsid" type="hidden" value="<?php echo $omcsid; ?>" />
 						<input name="tabindex" type="hidden" value="1" />
-						<input name="submitaction" type="submit" value="Edit Crowdsource Project" />
+						<button name="submitaction" type="submit" value="Edit Crowdsource Project" ><?php echo $LANG['EDIT_CROWD_PROJ']; ?></button>
```

Comment:
Why a button and not an input here?

Reply:
I think it's the only way to insert the php such that the text on the button says "Edit Crowdsource Project" (from the lang tag). Most things have been changed from inputs into buttons for this reason.